### PR TITLE
feat: check if user exists

### PR DESCRIPTION
**Pull request by schmelto**
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,37 +22,68 @@ function questions() {
...
-          .catch((err) => {
-            console.log(err);
-          });
+        checkUser(GHUsername).then((result) => {
+          if (result === true) {
+            questions(GHUsername);
+          } else {
+            console.log(
+              chalk.bgRed.bold(
+                ` User with username ${GHUsername} does not exist, try again! `
```
Comment (Pradumnasaraf) [inline src/index.js:33]:
```suggestion
                ` User with username ${GHUsername} does not exist on GitHub, try again! `
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you so much Tom for the PR. I was adding this check-in in the 1st place, though it will be a little strict to file naming. But, I think we should implement that from a long-term perspective.


Just minor changes are required so that the user can understand the prompt easily where the user name does not exit

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions, Tom :heart:

**Comment by schmelto** [comment]
Thanks!

**Pull request merged**